docs/Components-Toggle.md

```diff
@@ -0,0 +1,179 @@
+# Toggle Component
+
+[Back to all components](./Components.md)
+
+This component is a flexible component that can be used to toggle a boolean value. It can be used to render a switch, checkbox, or radio button, and exposes style props for every element.
+
+## Props
+
+### `variant`
+
+The `variant` prop determines the type of toggle to render. It can be one of the following: `switch`, `checkbox`, or `radio`.
+
+```tsx
+<Toggle value={value} onValueChange={setValue} variant="checkbox" />
+```
+
+### `status`
+
+The `status` prop is used to determine the interactibility or style of the toggle. It can be set to `disabled` or `error`. It is `null` by default.
```

Comment:
I don't think this matters for how the component will function, but technically the default is `undefined` since `null` is considered a value of its own.

![yulolimum-capture-2022-09-28--07-37-35](https://user-images.githubusercontent.com/1775841/192808458-8dc4a7d6-6921-4927-bb85-4c048a6587fa.jpg)